### PR TITLE
Heroku

### LINKED ISSUE #94: Fix Heroku
Screenshot from 2022-05-05 09-12-48
422 not good

### PR DESCRIPTION
**Pull request by jb3rndt**
Herokuuuuuu

Closes #94

### CODE REVIEW

File: api/mod/src/database/database.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 Base = declarative_base()
 
-engine = create_engine(SETTINGS.database_url, connect_args={"check_same_thread": False})
```
Comment (jb3rndt) [inline api/mod/src/database/database.py:13]: This option is incompatible with postgres. I tested around a bit and it doesnt seem to break anything locally. We might need to keep an eye on that

**Review by jb3rndt — COMMENTED** [review]
On Heroku, we will use a postgres instance. SQLite does not really work on heroku

File: api/mod/src/settings.py [hunk 2]
```diff
@@ -34,3 +34,7 @@ class Config:
 
 
 SETTINGS = Settings()
+if SETTINGS.database_url.startswith("postgres://"):
+    SETTINGS.database_url = SETTINGS.database_url.replace(
```
Comment (bewee) [inline api/mod/src/settings.py:38]: Why?

**Review by bewee — APPROVED** [review]
Just two things:
- Changes in the load_test_data script, but we do not use it in heroku afaics? 🤔 
- We could consider add a setting for that thread stuff which we set to false in production and keep it true for dev. But since you say everything works fine in dev anyway, I'd leave at it as it is and keep it in the back of our minds 👍

File: api/mod/src/settings.py (hunk 2 above)
Comment (jb3rndt) [reply api/mod/src/settings.py:38]: Heroku automatically sets a DATBASE_URL config variable that holds the connection string to the database. Sadly, it is not editable. But sqlalchemy will only recognise the database type when the string starts with `postgresql`, but Heroku writes only "postgres" so this needs to be replaced

**Comment by jb3rndt** [comment]
Yeah the changes to the load script are just for refactoring, which I recognized through doing this. I think its nothing very special so we can just keep it in here?

File: api/mod/src/settings.py (hunk 2 above)
Comment (bewee) [reply api/mod/src/settings.py:38]: okidoki

**Pull request merged**